walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,185 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
+import java.util.Collections;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    //If there are no events or no one is attending the event, 
```

Comment:
This seems like it should be a function comment (ie above the function like the javadoc comments) otherwise I would say the comments should be close to the code that's doing the things the comment is talking about, though you seem to have commented the rest of the function well.

Reply:
Done